### ISSUE TITLE
[Suggestion] Please reimplement throttle for Boost mode

### ISSUE THREAD

**Issue by Dwarf-LordPangolin** [tsunamayo/Skywanderers#1283]
There are two main reasons that throttle is necessary in Boost: the first relating to gameplay, and the second to quality-of-life.

The first is that there are many times that players will want to fly at the same speed as other ships that are slower. If I'm flying a fast fighter, and I'm escorting a friend who's flying a freighter, I need to be able to use less than 100% of my Boost speed -- otherwise I'll keep having to start and stop, or fly in loops so as not to run ahead of them.

Most importantly of all though, matching speed is _essential_ for combat. Fighters need to be able to slow down when they are on the tail of an enemy fighter, otherwise they'll either run into that enemy fighter, drop in and out of Boost repeatedly, or will keep having to break off, losing their advantage. A fighter attacking a ground target may want to slow down so as not to overshoot their target before they can target it properly (and to lessen the risk of running into the ground). Large ships in turn need to be able to adjust their speed to make sure that they don't pass other capital ships too quickly while firing on them with turrets. There are other examples, but these are some of the more-common ones.

The second reason is related to QOL: if the player wants to fly at low speeds currently, they need to do so in the 6DOF mode, which requires them to constantly press down the W key. This becomes annoying fairly quickly, and after a while your finger starts to cramp. There are going to be numerous instances where the player wants to fly slower than Boost's top speed, but not wish to do so at 6DOF speeds, or will want to have hands free while doing so.

**Comment by Tommyg930** [comment]
seconded!

**Comment by ZachZent** [comment]
The ability to change thrust used to be a feature, but disappeared with an update like a year ago. Was it disabled accidentally?

**Comment by tsunamayo** [comment]
Yes I will make further progress and gameplay, see what is needed and how I can implement everything nicely into a cohesive control scheme. I just need to have proper combat first and see how things turns out. I keep you posted, it will require a lot of testing.
